src/sentry/conf/server.py

```diff
@@ -3958,6 +3958,9 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 SEER_GROUPING_RECORDS_URL = (
     f"/{SEER_SIMILARITY_MODEL_VERSION}/issues/similar-issues/grouping-record"
 )
+SEER_GROUPING_RECORDS_DELETE_URL = (
+    f"/{SEER_SIMILARITY_MODEL_VERSION}/issues/similar-issues/grouping-record/delete/"
```

Comment:
you're concatenating that with `"/{project_id}"` so no need for a trailing slash here

```suggestion
    f"/{SEER_SIMILARITY_MODEL_VERSION}/issues/similar-issues/grouping-record/delete"
```

side note: why isn't this just `SEER_GROUPING_RECORDS_URL` with a verb `DELETE`?

Reply:
the seer records are done in a per-batch fashion per N groups, while this method is simply taking a single project_id and performing all the deletions.